### PR TITLE
Docs - Introduce links that are navigable when browsing the repo

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 
Second part of #311

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
WIP is over. Welcome to take a look.

File: docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ Mozilla Firefox   | `ff`
...
 
-You can use these aliases when working both through the [command line](/testcafe/documentation/using-testcafe/command-line-interface/#local-browsers)
-and [programmatically](/testcafe/documentation/using-testcafe/programming-interface/Runner/#browsers).
+You can use these aliases when working both through the [command line](../command-line-interface.md#local-browsers)
+and [programmatically](../programming-interface/Runner.md#browsers).
 
 To obtain the list of browsers installed on the current machine, run TestCafe
-from the command line with the [-b option](/testcafe/documentation/using-testcafe/command-line-interface/#b-list-browsers).
\ No newline at end of file
+from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md]: `../command-line-interface.md#-b---list-browser` This link still doesn't work

**Comment by inikulin** [comment]
\r-

File: docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md (hunk 1 above)
Comment (VasilyStrelyaev) [inline docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md]: works pretty fine for me

Comment (inikulin) [inline docs/articles/documentation/using-testcafe/common-concepts/browser-aliases.md]: Yep, something wrong with browser

**Comment by VasilyStrelyaev** [comment]
\ping @churkin @kirovboris @MargaritaLoseva @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
lgtm

**Pull request merged**